MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/ColliderPatch.cs

```diff
@@ -10,13 +10,13 @@ namespace MixedRealityExtension.Patching.Types
 	public class ColliderPatch : IPatchable
```

Comment:
>ColliderPatch [](start = 14, length = 13)

This class should have docstrings, or make it internal.

Reply:
All the patch classes are currently public and undocumented. Let's file a housekeeping bug.

---
In reply to: [335215069](https://github.com/microsoft/mixed-reality-extension-unity/pull/119#discussion_r335215069) [](ancestors = 335215069)